src/ProgressOnderwijsUtils/Collections/Maybe.cs

```diff
@@ -77,18 +86,17 @@ public TOut Extract<TOut>(Func<TOk, TOut> ifOk, Func<TError, TOut> ifError)
         public static implicit operator Maybe<TOk, TError>(Maybe_Ok<TOk> err)
             => new Maybe<TOk, TError>(err);
 
-        public bool TryGet(out TOk okValueIfOk, out TError errorValueIfError)
+        public bool TryGet([MaybeNullWhen(false)] out TOk okValueIfOk, [MaybeNullWhen(true)] out TError errorValueIfError)
         {
             switch (okOrError) {
                 case Maybe_Ok<TOk> okValue:
-                    (okValueIfOk, errorValueIfError) = (okValue.Value, default(TError));
+                    (okValueIfOk, errorValueIfError) = (okValue.Value, default(TError)!);

```

Comment:
Deze annotatie klopt toch niet?

Reply:
Nee, maar wat doen we eraan?

Reply:
tuple assignment splitsen?

Reply:
(geen idee of dat helpt natuurlijk :-) )

Reply:
anders iig een comment weer